content/chat/rooms/history.textile

```diff
@@ -0,0 +1,59 @@
+---
+title: Message storage and history
+meta_description: "Retrieve previously sent messages from history."
+product: chat
+languages:
+  - javascript
+---
+
+The history feature enables users to retrieve messages that have been previously sent in a room. Ably stores chat messages for 24 hours by default. You can be extend this up to 30 days by "contacting us":https://forms.gle/SmCLNFoRrYmkbZSf8.
+
+h2(#get). Retrieve previously sent messages
+
+Use the "@messages.get()@":https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/Messages.html#get method to retrieve messages that have been previously sent to a room:
+
+```[javascript]
+const historicalMessages = await room.messages.get({ direction: 'backwards', limit: 50 });
+console.log(historicalMessages.items);
+
+if (historicalMessages.hasNext()) {
+  const next = await historicalMessages.next();
+  console.log(next);
+} else {
+  console.log('End of messages');
+}
+```
+
+The following optional parameters can be passed when retrieving previously sent messages:
+
+|_. Parameter |_. Description |
+| start | Earliest time in milliseconds since the epoch to retrieve messages from.  |
```

Comment:
Subjective again, a little unclear to me. Can we rephrase to sound more like the docstring (but also mention that it's a unix timestamp in ms)? https://sdk.ably.com/builds/ably/ably-chat-js/main/typedoc/interfaces/chat_js.QueryOptions.html#start

(far fetched, I know, but is there a way to force this to fetch the docstring directly, and we should make the docstring more clear?)